packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -504,6 +518,19 @@ module.exports = React.createClass({
            fixtureName === this.props.fixture;
   },
 
+  userHasComponents() {
+    return Object.keys(this.props.fixtures).length > 0;
+  },
+
+  userHasFixtures() {
+    return _.reduce(this.props.fixtures, (acc, compFixtures) => {
+      const fixtureNames = Object.keys(compFixtures);
+      const noAutoFixture = fixtureNames[0].indexOf('(auto)') === -1;
+
+      return acc || (fixtureNames.length > 1 && noAutoFixture);
```

Comment:
Can't really use `for of` because it uses `Symbol` and PhantomJS does not support it. I guess we could polyfill it but how about just doing:

```
userHasFixtures() {
  return _.reduce(this.props.fixtures, (acc, compFixtures) => {
    const fixtureNames = Object.keys(compFixtures);

    return acc || fixtureNames[0].indexOf('(auto)') === -1;
  }, false);
},
```